### DISCUSSION TITLE
Cycle plan until 2023-06-22

### DISCUSSION THREAD

**Discussion by onOffice-Web** [onOffice-Web-Org/oo-wp-plugin#543] [Announcements]
We decided to bet on the following pitches for this cycle:

- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/542
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/541
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/540
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/539
- https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/538

The cycle goes until Tue 2023-06-22, inclusive.

For any questions, don't hesitate to ask me via chat or in the weeklies.

**Comment by fredericalpers** [comment]
@yeneastgate @dai-eastgate I prepared the next cycle plan already, since theres a public holiday coming up again.

**Reply by dai-eastgate** [reply]
> @yeneastgate @dai-eastgate I prepared the next cycle plan already, since theres a public holiday coming up again.

Ok, we got it, thanks.
